app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,83 @@
+class InvitationsController < ApplicationController
+  before_action :authenticate_with_pre_login_destination, only: [:show]
+  before_action :set_organization, only: [:create]
+
+  def create
```

Comment:
This method has a lot of conditionals. Watch that talk I sent you and see if you can prefer sending messages rather than using conditionals. Tell, don't ask.